content/rule_book/roles_and_responsibilities.md

```diff
@@ -2,68 +2,96 @@
-    - If an explanation of a ruling is requested by a captain, the quizmaster will ask if any team wants to challenge before the explanation is given.
+- Rule immediately upon an obviously correct or incorrect answer if there is no indication from the answer judge to the contrary
+
+When a quizzer's response contains a required completion of the question and that completion is different than the written question, the quizmaster will discuss it with the answer judge. If the quizzer gives a reference question that is different than the written question, the quizmaster and answer judge must ensure that the reference question given is the exact same type as the written question.
+
+The quizmaster should consult with the answer judge(s) at any time they feel is necessary, provided it does not infringe upon the quizzer's question answer duration period.
+
+#### Quizmaster Prompts
+
+During the question answer duration period, the quizmaster will use only the following prompts.
+
+##### Pronoun Clarification
+
+If a quizzer has provided all the information in the answer, except the identification of a pronoun present which is part of the answer text, the quizmaster will ask the quizzer to identify the specific pronoun. For example: "Can you identify 'He'?" or "Can you identify 'They'?"
+
+The quizmaster is not allowed to say anything other than what the quizzer has already said when asking for the identification. A pronoun may need to be identified only if the antecedent is in context.
+
+Once the quizzer has completed all information in the question and answer, the quizmaster will prompt the quizzer to identify the specific pronoun.
+
+##### Reference Questions
+
+On a Reference type question, if the quizzer has given all the information in both the question and the answer and the complete question was not read, the quizmaster will ask, "What is your question?"
+
+##### Answer Correction
+
+On a Quote/Finish type question, once the quizzer has quoted to the end of the required verse or verses, if the quote is not yet word-perfect, the quizmaster will say, "Again." This will occur as often as needed until the quizzer quotes the verse perfectly or the question answer duration period expires. The quizmaster is not allowed to say "again" until the quizzer has come to the end of the verse, each time through, as this would give the quizzer information about where their mistake was made.
+
+For all other question types, once all of the information in both the question and answer has been given, but something needs to be fixed, the quizmaster will say "Again".
+
+##### Quote is Complete
+
+On a Situation type question, when the quizzer has completed the quotation, the quizmaster will say, "Quote is complete."
+
+##### Appending an Answer
+
+On an INT or MA Question, if all the information in the answer text has been given but the quizzer is missing information in the question text, the quizmaster will say "More".
```

Comment:
caps on "Question?"  (help me out with the pattern/rule you're following)

Reply:
That's a punctuation bug. Patched in 977dbb6